### DISCUSSION TITLE
Request to domain name blocked from Nigeria

### DISCUSSION THREAD

**Discussion by Iyin0** [vercel/community#6037] [Help]
### Question

Hello,

Requests from browsers in Nigeria to the domain linked main urls are being blocked. I cannot access "leytro.com" or "quickly.com.ng"  from Nigeria but I can access the subdomain linked to them. I should also note that the websites is accessible from other countries. I am experiencing this same issue on multiple projects on another account. 

The domain name has been linked since last year and was working up until I noticed on 5th March, 2024.

**Comment by amyegan** [comment]
Thanks for reaching out @Iyin0. Are you still experiencing this problem?

**Reply by Iyin0** [reply]
Yes, I am

**Reply by amyegan** [reply]
Thanks for reporting this. Your site is not the only one affected. 

We reached out to the ISP to get these sites unblocked, but don't have much control over when that will happen. Thank you for your patience while we work with them to get this solved. 🙇

**Reply by Iyin0** [reply]
Okay. Will be expecting your response once it has been solved

**Reply by otolo2024** [reply]
i am experiencing this problem too for days now
